package.json

```diff
@@ -33,10 +33,10 @@
 		"node": ">=14"
 	},
 	"dependencies": {
-		"cached-hafas-client": "^4.0.0",
```

Comment:
The `^` in the value expresses a SemVer range, so this change is not relevant. (The actual version of a dependency being installed is determined by the lockfile if present.)

In fact, non-major-upgrades are a bit like noise, e.g. when I want to `git blame`, so please remove this. Thanks!

Reply:
Fixed by removing the commit where I changed that. So it doesnt clutter up the history